templates/base.html

```diff
@@ -0,0 +1,72 @@
+{% load staticfiles %}
+{% load compress %}
+<!doctype html>
+<html class="no-js" lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    {% block meta %}{% endblock meta %}
+    {% block base_title %}
+    <title>{% block title %}{% endblock title %} - OpenCraft -</title>
+    {% endblock base_title %}
+
+    {# Base Foundation CSS #}
+    {% block foundation_css %}
+    {% compress css %}
+    <link href='{% static "foundation/scss/normalize.scss" %}' type="text/x-scss" rel="stylesheet"/>
+    <link href='{% static "foundation/scss/foundation.scss" %}' type="text/x-scss" rel="stylesheet"/>
+    {% endcompress %}
+    {% endblock foundation_css %}
+
+    {# Custom CSS #}
+    {% block css %}{% endblock css %}
+
+    {# Underscore #}
+    {% block underscore_js %}
+    <script src="//cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore-min.js"></script>
+    {% endblock underscore_js %}
+
+    {# Modernizr #}
+    {% block foundation_modernizer %}
+    <script src='{% static "foundation/js/vendor/modernizr.js" %}'></script>
+    {% endblock foundation_modernizer %}
+
+    {# Angular #}
+    {% block angular_js %}
+    <script src="//ajax.googleapis.com/ajax/libs/angularjs/1.4.3/angular.js"></script>
```

Comment:
Nit: you should use `angular.min.js` which is available in the same location, and includes a `.map` file so it should be no worse for debugging.